### PR TITLE
feat: [STO-5103] Add CardVariant types needed by STO MFE

### PR DESCRIPTION
**Pull request by chris-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/microfrontends/index.ts [hunk 1]
```diff
@@ -45,6 +46,11 @@ import type { ChaosCustomMicroFrontendProps } from '@chaos/interfaces/Chaos.type
 import type { FFCustomMicroFrontendProps } from '@cf/FFCustomMicroFrontendProps.types'
 import type { getLocationPathName } from 'framework/utils/WindowLocation'
 
+// Service types
+export * as cdNg from 'services/cd-ng'
+export * as ci from 'services/ci'
+export * as pipelineNg from 'services/pipeline-ng'
```
Comment (abhinavrastogi-harness) [inline src/microfrontends/index.ts:52]: we should not be re-exporting all services from microfrontends package. it causes a lot of bloat, for types which might not be needed by other microfrontends.

Few options that might be better imo:
1. Pass them as custom props only to your microfrontend
2. Pass only the keys that are required right now
3. Wait for the [api clients](https://github.com/harness/react-api-client) to be published as separate packages which @sunnykesh-harness is working on. It should be done in 1-2 weeks. Those can be added as direct dependencies in your microfrontend

**Comment by chris-harness** [comment]
retrigger cypress-cv

**Review by ahokinson — APPROVED** [review]

File: src/microfrontends/index.ts [hunk 2]
```diff
@@ -113,5 +114,6 @@ export {
   CCMUIAppCustomProps,
   TIUIAppCustomProps,
   ChaosCustomMicroFrontendProps,
-  FFCustomMicroFrontendProps
+  FFCustomMicroFrontendProps,
+  CardVariant
```
Comment (knagurski) [inline src/microfrontends/index.ts:118]: Instead of exporting this here, add it to your own MFE type def and pass it through as a custom prop

Comment (chris-harness) [reply src/microfrontends/index.ts:118]: That really slows down development, though, as we need to wait for a NG UI release before we can merge the dependent changes.

**Pull request closed without merging**